packages/core-data/src/entities.js

```diff
@@ -207,6 +209,7 @@ export const rootEntitiesConfig = [
 		baseURLParams: { context: 'edit' },
 		plural: 'plugins',
 		key: 'plugin',
+		supportsPagination: true,
```

Comment:
Hmm, might have to remove this one: 

https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-plugins-controller.php#L1000

Reply:
Yes, the plugins endpoint doesn't seem to support this - https://developer.wordpress.org/rest-api/reference/plugins/.